Documentation/getting-started.md

```diff
@@ -0,0 +1,112 @@
+# Adobe Experience Places Service
+
+Adobe Experience Platform Location Service provides an SDK extension which allows you to act based on the location of your users. This extension is the interface to the [Location Service Web Services APIs](https://experienceleague.adobe.com/docs/places/using/web-service-api/places-web-services.html?lang=en).
+
+The SDK extension listens for events that contain GPS coordinates and geofence region events, and dispatches new events that are processed by the Rules Engine. The SDK extension also retrieves and delivers a list of the nearest POI for the app data that retrieves from the APIs. The regions returned by the APIs are stored in cache and persistence, which allows limited offline processing.
+
+`Places` is the mobile SDK supporting the Location Service.
+
+## Configure the Places extension in Data Collection UI
+
+1. In the Data Collection UI, from your mobile property, select the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Places** extension, and select **Install**.
+3. Select the **POI Library (or libraries)** you wish to use in the app.
+4. Select **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+![Places extension configuration](./assets/config.png)
+
+## Add the AEP Places extension to your app
+
+## Download and import the Places extension
+
+1. Add the Mobile Core and Places extensions to your project using the app's Gradle file.
+
+```java
+implementation 'com.adobe.marketing.mobile:core:2.+'
+implementation 'com.adobe.marketing.mobile:places:2.+'
+```
+
+> **Warning**
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/GradleDependencies.md) for managing gradle dependencies.
```

Comment:
`https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/GradleDependencies.md` link needs to be changed :(